libs/stac/odc/stac/_load.py

```diff
@@ -28,17 +28,41 @@ def load(
     stac_cfg: Optional[ConversionConfig] = None,
     patch_url: Optional[Callable[[str], str]] = None,
     **kw,
-) -> xr.Dataset:
+) -> xarray.Dataset:
     """
-    Load STAC Items (from the same or similar collections) as an
-    xarray Dataset.
+    Load several STAC :class:`pystac.Item` objects (from the same or similar
+    collections) as an :class:`xarray.Dataset`
+
+    This method can load pixel data directly locally or construct Dask graph that can
+    be processed on a remote cluster.
+
+    .. code-block:: python
+
+       catalog = pystac.Client.open(...)
+       query = catalog.search(...)
+       xx = odc.stac.load(
+           query.get_items(),
+           bands=["red", "green", "blue"],
+           output_crs="EPSG:32606",
+           resolution=(100, -100),
+       )
+       xx.red.plot.imshow(col="time");
+
+
+    .. note::
+
+       At the moment one must specify desired projection and resolution of
+       the result. The plan is to make this choice automatic if not configured.
+
+
+    **Parameters**
 
     :param items:
-       STAC Items to load
+       Collection of STAC :class:`stac.Item` to load
```

Comment:
```suggestion
       Iterable of STAC :class:`stac.Item` to load
```

`Collection` is a noun with specific meaning in the STAC ecosystem.

Reply:
Good catch, maybe `Sequence` is a more "human" name for it rather than `Iterable`? Although in Python `Sequence` is basically a kind of `Iterable` the one that supports `len()` and `[idx]`.

Reply:
Since it's API documentation, IMO `Iterable` is ok to use because presumably only technical users will care about this function.